include/fmt/base.h

```diff
@@ -539,7 +539,7 @@ template <typename Char> class basic_string_view {
 #endif
   FMT_CONSTEXPR20 basic_string_view(const Char* s) : data_(s) {
 #if FMT_HAS_BUILTIN(__builtin_strlen) || FMT_GCC_VERSION || FMT_CLANG_VERSION
-    if (std::is_same<Char, char>::value) {
+    if (std::is_same<Char, char>::value && !detail::is_constant_evaluated()) {
       size_ = __builtin_strlen(detail::narrow(s));
```

Comment:
What's the advantage of still using `__builtin_strlen`? Will gcc or clang still end up producing a constant in some scenarios that they wouldn't if just `strlen` was used?

Reply:
Well... libstdc++ __builtin_strlen in char_traits, so I guess there's that.

Reply:
There was some benefit to the debug codegen at some point but probably no longer matters on newer compilers.

Reply:
More importantly, `strlen` is not constexpr.